kotest-framework/kotest-framework-engine/src/jvmMain/kotlin/io/kotest/engine/KotestEngine.kt

```diff
@@ -49,6 +50,7 @@ class KotestEngine(private val config: KotestEngineConfig) {
     * Starts execution of the given test plan.
     */
    suspend fun execute(plan: TestPlan) {
+      val projectExtensions = configuration.extensions().filterIsInstance<ProjectExtension>().reversed()
```

Comment:
We don't need to reverse - we simply won't make any guarantees on the order they run.